.github/workflows/cr.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Any Commit
+on:
+  push:
+    branches:
+      - main
+      - v6/environment-api
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  build:
+    if: github.event_name == 'push' || github.event.review.state == 'APPROVED'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - run: corepack enable
```

Comment:
Can we do something like this instead to be consistent with our current setups?

https://github.com/vitejs/vite/blob/feae09fdfab505e58950c915fe5d8dd103d5ffb9/.github/workflows/ci.yml#L70-L72